src/Gui/Action.cpp

```diff
@@ -529,18 +529,18 @@ void WorkbenchGroup::addTo(QWidget *w)
 {
     refreshWorkbenchList();
     if (w->inherits("QToolBar")) {
-        QToolBar* bar = qobject_cast<QToolBar*>(w);
-        QComboBox* box = new WorkbenchComboBox(this, w);
+        auto *bar = qobject_cast<QToolBar *>(w);
+        QComboBox *box = new WorkbenchComboBox(this, w);

```

Comment:
Why are you randomly moving the star aside the type or aside the variable? I could understand such moves to achieve consistency, but it's not the case across this PR.

Reply:
Thankyou OpenBrain. Within the FreeCAD code base the position of the * is not consistent. Of course c++ doesn't give a rat's bum where it is. I have been progressively using auto formatting in my IDE (Clion) using the clang-format file recently introduced into FreeCAD in an attempt to achieve consistency. Apologies if I havn't reformatted all touched code areas. If the question is whether the * should be to the left or the right by all means lets have the discussion to set the standard going forward. If there exists some FreeCAD coding guidelines please advise. I have not seen any.

Reply:
Why auto for the former and not the later type ? It there a reason using box as a `QComboBox` wouldn't work as a `WorkbenchComboBox` later on ? It doesn't seem consistent.

Reply:
The first case states specifically object_cast<QToolBar*> so the type is obvious. No need to repeat (DRY).
The second case is new WorkbenchComboBox but the type is stated as QComboBox. Without knowing or caring why, there is a difference, so not a direct candidate for auto.